docs/source/api/pipelines/stable_diffusion/depth2img.mdx

```diff
@@ -0,0 +1,21 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Depth-to-Image Generation
```

Comment:
Could we maybe add a couple of sentences here where this pipeline comes from ? 
E.g. copy the relevant information: "Who first defined the pipeline", "What does the pipeline do", "Most important public checkpoint" , "Original codebase"